### PR TITLE
Feature/deterministic test vector

### PR DESCRIPTION
**Pull request by xaviaracil**
New section with a complete example of the creation of a Linked Data Proof

### CODE REVIEW

**Comment by xaviaracil** [comment]
On hold waiting for a reorganization of the chapters, with the addition of new chapters for:
- Issuer
- Holder
- Verifier

File: ob_v3p0/impl/linked-data-proof.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    222, 175, 222,  46, 168, 190, 239, 173, 216, 198, 153,
+    181, 199, 224, 112,  76, 245,  17,  84, 213,  46,  23,
+    178,  11, 113,  51, 124, 160,  76, 197, 165
+  ],
+  publicKeyMultibase: 'z6MkjZRZv3aez3r18pB1RBFJR1kwUVJ5jHt92JmQwXbd5hwi',
+  secretKeyMultibase: 'zrv2bqTbNwCTsRrHFcJCPjVAduh4Ezcnoq1A3ZxH1GWTNkxipLVuaAoMFmze2gFN9oNXfJjufxSHWVZzsJiUsMHFMcx',
+  revoked: undefined,
+  export: [AsyncFunction: export],
+  signer: [Function: signer],
+  verifier: [Function: verifier]
```
Comment (ottonomy) [inline ob_v3p0/impl/linked-data-proof.md:36]: Is this intended to be double-quoted valid JSON? Doesn't quite look like it. Maybe the annotation could be changed to `js` instead to enable proper syntax highlighting.

Comment (xaviaracil) [reply ob_v3p0/impl/linked-data-proof.md:36]: Sorry @ottonomy, you're right. This code is in js, not json

**Comment by xaviaracil** [comment]
As agreed in previous meetings I want to differentiate role-related information in the implementation guide. Therefore, any Issuer, Displayer or Host can easely find out their related information.

One approach to that is to have a first level section for each role and then have subsections (like `Quickstart` and `Best Practices`) for each of them, ie:
- Issuer
  -  Quickstart
  - Best Practices
- Displayer
  -  Quickstart
  - Best Practices
- Host
  -  Quickstart
  - Best Practices

The other approach is making these role as subsections of current sections, ie:
- Quickstart
  - Issuer Quickstart
  - Displayer Quickstart
  - Host Quickstart
  - API Quickstart (as is related to Issuer and Host)
- Best Practices
  - Common
  - Issuer 
  - Displayer
  - Host

I personally would go for the second option, but I'm open to suggestions.

**Pull request merged**